design/apidsl/action.go

```diff
@@ -474,6 +475,19 @@ func payload(isOptional bool, p interface{}, dsls ...func()) {
 			TypeName:            fmt.Sprintf("%s%sPayload", an, rn),
 		}
 		a.PayloadOptional = isOptional
+		a.PayloadMultipart = isMultipart
+	}
+}
+
+// MultipartForm can be used in: Action
+//
+// MultipartForm implements the action multipart form DSL. An action multipart form indicates that
+// the HTTP request body should be encoded using multipart form data as described in
+// https://www.w3.org/TR/html401/interact/forms.html#h-17.13.4.2.
+//
+func MultipartForm(p interface{}, dsls ...func()) {
```

Comment:
Do we really need arguments to `MultipartForm`?

Reply:
It's my mistake. https://github.com/goadesign/goa/pull/1582/commits/d4f73d7acc8a2388ab1b402c69667c43ae50c6a0 removes aruments of `MultipartForm`.